pages/index.html

```diff
@@ -4,9 +4,55 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="/styles/styles.css" rel="stylesheet">
+  <title>Homepage</title>
 </head>
 <body>
-  
+  <section class="container">
+    <div class="header">
+      <header>
+        <h1>Arika Agnew's Personal Portfolio</h1>
+      </header>
+    </div>
+
+    <div class="main-body">
```

Comment:
I can tell from the class name of this div that the content that follows is the "main part" of this page. Since that's the case, consider using a <main> tag to help add semantic structure to your page